service/src/main/resources/db/changesets/20230608_schema_common_reset.yaml

```diff
@@ -0,0 +1,693 @@
+databaseChangeLog:
+  - changeSet:
+      id: schema_reset
+      author: marikomedlock, chenchalsubraveti
+      dbms: mariadb,mysql
+      changes:
+        - createTable:
+            tableName: study
+            columns:
+              - column:
+                  name: id
+                  type: ${id.type}
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+                    unique: true
+              - column:
+                  name: display_name
+                  type: ${text.type}
+                  constraints:
+                    nullable: true
+              - column:
+                  name: description
+                  type: ${text.type}
+                  constraints:
+                    nullable: true
+              - column:
+                  name: created
+                  type: ${date.timezone}
+                  constraints:
+                    nullable: false
+                  defaultValueComputed: now()
+              - column:
+                  name: created_by
+                  type: ${text.type}
+                  constraints:
+                    nullable: false
+              - column:
+                  name: last_modified
+                  type: ${date.timezone}
+                  constraints:
+                    nullable: false
+                  defaultValueComputed: now()
+              - column:
+                  name: last_modified_by
+                  type: ${text.type}
+                  constraints:
+                    nullable: false
+              # needed for test cases to run, when removing this columns
+              # fix test cases in StudyServiceTest
+              - column:
+                  name: properties
+                  type: ${jsonb.type}
+                  constraints:
+                    nullable: true
```

Comment:
Ah I think this is my bug -- leftover from when I replaced the properties column with a separate table. I think you can just remove "properties" from [this line](https://github.com/DataBiosphere/tanagra/blob/6e60dacabdab989ea71183f5accc0d2c0da476be/service/src/main/java/bio/terra/tanagra/db/StudyDao.java#L29) to fix, and then you won't need to keep this or the `jsonb.type` around.

Reply:
Remove this initially, and specifically broke I think the study tests as it needed fiields(?) that is why I put it back. Once the specific tests are fixed, we can drop this column. The other text[] columns are removed and did not affect test cases